screens/EpisodeInputFlow/EpisodeDisplayScreen.tsx

```diff
@@ -1,45 +1,104 @@
-import React from "react";
-import { StyleSheet, ScrollView } from "react-native";
+import React, { useEffect, useRef, useState } from "react";
+import {
+  StyleSheet,
+  ScrollView,
+  useWindowDimensions,
+  Animated,
+  TouchableOpacity,
+} from "react-native";
 import { Text, View } from "../../components/Themed";
 import { Episode } from "../../types";
 import { convertToLocale } from "../../utils/TimeUtils";
+import { Ionicons } from "@expo/vector-icons";
+import Colors from "../../constants/Colors";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { STORAGE_KEYS } from "../../utils/AsyncStorageUtils";
+import { useNavigation } from "@react-navigation/native";
 
 interface EpisodeProps {
   episode: Episode;
   index: number;
+  deleteCallback: () => void;
 }
 
-function EpisodeDisplay({ episode, index }: EpisodeProps) {
+function EpisodeDisplay({ episode, index, deleteCallback }: EpisodeProps) {
+  const [clicked, setClicked] = useState(false);
+  const { width } = useWindowDimensions();
+  const offset = useRef(new Animated.Value(0)).current; // Initial value for opacity: 0
+
+  useEffect(() => {
+    const toValue = clicked ? -width + 60 : 0;
+    Animated.timing(offset, {
```

Comment:
Not sure if there's anything that can be done about it or if it's just how it shows up on my phone but the animation for deleting looks pretty blocky (and sometimes doesn't really animate at all)

Reply:
oof, will take a look, thanks

Reply:
should be fixed now!